orbi_dnsmasq/__main__.py

```diff
@@ -21,14 +29,30 @@
     help="Toggle telnet service on the orbi debug site with selenium"
 )
 parser.add_option(
-    "-w",
-    "--webdriver",
+    "-a",
+    "--address",
+    action="store",
+    default="orbilogin.com",
+    dest="address",
+    help="IP or hostname of your router",
+    type="string"
+)
+parser.add_option(
+    "-p",
+    "--password",
     action="store",
     default=None,
-    dest="webdriver_path",
-    help="Path to the webdriver for selenium",
+    dest="password",
+    help="The password for the admin user",
```

Comment:
Hey, I don't really personally like using this sort of params, as your password will be recorded in your terminal's history. Nevertheless, I do agree that it's pretty convenient, so as a middle ground I propose changing this description:

```suggestion
    help="The password for the admin user (warning: insecure, watch out for your terminal's history)",
```

What do you think?